docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+        <i class="fa fa-fire fa-lg" style="color:#7e0038"></i>
+        <i class="fa fa-fire fa-lg" style="color:lightgrey"></i>
+        <i class="fa fa-fire fa-lg" style="color:lightgrey"></i>
+  <!--       <p><span data-v-013baba1="" title="" class=""><svg data-v-013baba1="" viewBox="0 0 16 16" width="1em" height="1em" focusable="false" role="img" alt="icon" xmlns="http://www.w3.org/2000/svg" fill="currentColor" class="bi-bar-chart-fill b-icon bi medium-level"><g data-v-013baba1=""><rect width="4" height="5" x="1" y="10" rx="1"></rect><rect width="4" height="9" x="6" y="6" rx="1"></rect><rect width="4" height="14" x="11" y="1" rx="1"></rect></g></svg> Medium </span></p> -->
+      </div>
+    </div>
+  </div>  
+  <div class="column">
+    <div class="card">
+      <div class="container">
+        <i class="fa fa-clock-o fa-2x" style="color:#7e0038;"></i>
+        <h4><b>Reading Time</b></h4>
+        <p><i class="fa fa-clock-o fa-lg" style="color:#7e0038;"></i> 30 minutes</p>
+        <h4><b>Recipe Type</b></h4>
+        <p><i class="fa fa-laptop fa-lg" style="color:#7e0038;"></i> Background Information</p>
+        <h4><b>Executable Code</b></h4>
+        <p><i class="fa fa-play-circle fa-lg" style="color:#7e0038;"></i> Minimal</p>
+      </div>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <div class="container">
+        <i class="fa fa-group fa-2x" style="color:#7e0038;"></i>
+        <h4><b>Intended Audience</b></h4>
+        <p> <i class="fa fa-user-md fa-lg" style="color:#7e0038;"></i> Principal Investigators </p>
+        <p> <i class="fa fa-database fa-lg" style="color:#7e0038;"></i> Data Managers </p>
+        <p> <i class="fa fa-wrench fa-lg" style="color:#7e0038;"></i> Data Scientists </p>
+<!--         <p> <i class="fa fa-money fa-lg" style="color:#7e0038;"></i> Funders</p> -->
+      </div>
+    </div>
+  </div>
+</div>
+
+___
+
+
+The **FAIR principles**, under `Interoperability` state that: 
+> I3. (Meta)data include qualified references to other (meta)data 
+> 
+
+## Main Objectives
+
+The main goals of this recipe are:
+
+> - To understand where the need arises for mapping between data identifiers;
+> - How to represent equivalences for use by others; and
+> - Know what services are available to support data interoperability.
+
+This recipe assumes that you are already familiar with identifiers and the minting of identifiers.
+
+*For more details on identifiers, see the [How to generate globally unique, resolvable and persistent identifiers](https://fairplus.github.io/cookbook-dev/recipes/findability/identifiers.html) recipe.*
+
+
+## Graphical Overview of the FAIRification Recipe Objectives
+
+This recipe will cover the highlighted topics
+```mermaid
+    graph TB
+      I[Identifier]
+      I --> IM[Identifier Mapping]
+      M[Mapping] --> OM[Ontology Mapping]
+      OM --> OxO
+
+      M --> IM
+
+      IM -- has section --> IMS
+
+      IM -- has section--> IE([Identifier Equivalence])
```

Comment:
Check consistent whitespace around label of the edge. Maybe not technically important, but makes it easier to read.

Reply:
Graph updated